walker.go

```diff
@@ -36,15 +36,19 @@ func WalkWithContext(ctx context.Context, root string, walkFn func(pathname stri
 		return err
 	}
 
+	cpuLimit := runtime.NumCPU()
```

Comment:
Can you update this to use GOMAXPROCS? That's what it probably should have been using 🤦

Reply:
@lebauce pinging just in case this message gets missed

Reply:
Sorry :-( Completely missed the notification. Thank you for the merge !